components/scroller/index.js

```diff
@@ -0,0 +1 @@
+export { default as DtScroller } from './DtScroller.vue';
```

Comment:
Just to be consistent with the naming conventions of our other components, the file name should be 'Scroller.vue' but the export should be DtScroller

Reply:
Resolving after discussion due to issues with composition API and component names.